pom.xml

```diff
@@ -3,7 +3,7 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>1.609.1</version>
+		<version>3.25</version>
```

Comment:
The old parent pom referred to a dependency that wasn't in my contemporary Jenkins artifact repositories. In the interest of hygiene, I decided to see if the current plugin `pom.xml` parent would work. It did! Only required a few (also hygiene-related) changes to the pom; see below.